mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/concurrency/ConcurrentTestRunnerTest.java

```diff
@@ -120,18 +118,18 @@ public void consume(int threadNumber, int step) throws Exception {
     public void runShouldPerformAllOperations() throws Exception {
```

Comment:
What is this code for. I am not speaking about your modification but about the aim of this class, if this can be explained fastly ?